### ISSUE TITLE
MsSql

### ISSUE THREAD

**Issue by kivervinicius** [taviroquai/db2graphql#17]
Plan to support MSSQL?

**Comment by taviroquai** [comment]
Hi!

I think it is pretty possible to support MSSQL.

I will try to find some time to develop this.

It will go into [adapters](https://github.com/taviroquai/db2graphql/tree/master/src/adapters)

Thank you so much.

**Comment by kivervinicius** [comment]
Can I try to assist this process, basically if I take an adapter of this example and convert it to MSSQL functions would it solve?

**Comment by taviroquai** [comment]
Yes its should be no problem. Let me know ir you need Helô. Thanks you!

**Comment by taviroquai** [comment]
Hi @kivervinicius ,

I took some time to work on this feature. It is now available since commit https://github.com/taviroquai/db2graphql/commit/fb558cbfb23ff45d042eb34db87cd4e89ad0e18e !

It is also available on NPM using ```npm install db2graphql@0.8.1```

I have used a local MS SQL Server (Express) instance to test it. All tests passed but unfortunately Travis CI does not support MsSql service to test it upstream.

I had hard time to configure access to MS SQL database (side note).
This config should work:
```json
{
  "client": "mssql",
  "debug": true,
  "connection": {
    "host": "localhost",
    "database": "db2graphql",
    "user": "db2graphql",
    "password": "",
    "trustServerCertificate": true,
    "dialectOptions": {
      "instanceName": "SQLEXPRESS"
    }
  },
  "exclude": []
}
```

Also, these are the dependencies that you need to use with this library.
```json
{
  "apollo-server": "^3.10.0",
  "db2graphql": "^0.8.1",
  "graphql": "^16.5.0",
  "knex": "^2.2.0",
  "tedious": "^15.0.0"
}
```
Some dependencies require node version >= 14.

Let me know if you can test it and give some feedback.

Kind regards

**Comment by kivervinicius** [comment]
Olá amigo, não consigo testar agora, mas em breve farei, por enquanto havia saído do meu escopo essa lib, porém ela continua muito interessante, parabéns pelo progresso.
